### PR TITLE
review test chapter

### PR DESCRIPTION
**Pull request by alessandrofelder**
just some typos and clarifications of things I found unclear

### CODE REVIEW

File: ch03tests/07CI.ipynb [hunk 1]
```diff
@@ -72,4 +72,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 1
-}
+}
```
Comment (ageorgou) [inline ch03tests/07CI.ipynb:75]: I don't know why these newlines were deleted (also at `08DiffusionExample.ipynb`)!

File: ch03tests/08DiffusionExample.ipynb [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "1. If second energy is lower, accept move.\n",
     "1. $\\beta$ is a parameter which determines how likely\n",
     "   the simulation is to move from a 'less favourable' situation to a 'more favourable' one.\n",
-    "1. Compute $P_0=e^{-\\beta (E_1 - E_0)}$ and $P_1$ a random number between 0 and 1,\n",
+    "1. Compute $P_0=e^{-\\beta (E_1 - E_0)}$ and $P_1$ a uniformaly distributed random number between 0 and 1,\n",
```
Comment (ageorgou) [inline ch03tests/08DiffusionExample.ipynb:25]:
```suggestion
    "1. Compute $P_0=e^{-\\beta (E_1 - E_0)}$ and $P_1$ a uniformly distributed random number between 0 and 1,\n",
```
but yes!

**Review by ageorgou — CHANGES_REQUESTED** [review]
Good catches! Just a small typo and some newlines to stop git complaining...

File: ch03tests/07CI.ipynb (hunk 1 above)
Comment (dpshelio) [inline ch03tests/07CI.ipynb:75]:
```suggestion
}

```

File: ch03tests/08DiffusionExample.ipynb [hunk 3]
```diff
@@ -70549,4 +70549,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 1
-}
+}
```
Comment (dpshelio) [inline ch03tests/08DiffusionExample.ipynb:70552]:
```suggestion
}

```

**Review by dpshelio — APPROVED** [review]
There you go 😁

File: ch03tests/08DiffusionExample.ipynb (hunk 3 above)
Comment (ageorgou) [inline ch03tests/08DiffusionExample.ipynb:70552]:
```suggestion
}

```

**Review by ageorgou — APPROVED** [review]

**Pull request merged**